port/ztpf/omrvmem.c

```diff
@@ -1445,6 +1445,7 @@ getMemoryInRangeForDefaultPages(struct OMRPortLibrary *portLibrary,
 					PPG_vmem_pageSize[0], category);
 		}
 
+#ifndef OMRZTPF
```

Comment:
A comment should be added to highlight the key points from https://github.com/eclipse/omr/pull/6691#issuecomment-1259603661: mmap differences on z/TPF architecture, infinite loop issue and plans (TODOs) for the future.